docs/changelog.rst

```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+5.0.0
+-----
+
+- Add :meth:`gidgethub.routing.Router.fetch` for obtaining a frozenset of functions
+  registered to the router that the event would be called on.
+  (`Issue #74 <https://github.com/brettcannon/gidgethub/issues/74>`_).
```

Comment:
```suggestion
- Add :meth:`gidgethub.routing.Router.fetch` for obtaining a frozenset of functions
  registered to the router that the event would be called on.
  (`Issue #74 <https://github.com/brettcannon/gidgethub/issues/74>`_).
- Make router callback execution order non-deterministic to avoid relying on
   registration order.
   (`Issue #74 <https://github.com/brettcannon/gidgethub/issues/74>`_).
```